sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -142,7 +142,7 @@ class ShareSmbSettings(GeneratedShareSmbSettings):
 
     :param SmbMultichannel multichannel: Required. Sets the multichannel settings.
     """
-    def __init__(self, multichannel):
+    def __init__(self, multichannel, **kwargs):
```

Comment:
move multichannel into kwargs

Reply:
Why should it not currently be a required argument?

Reply:
Because later if ShareSmbSettings has more attr, multichannel is not necessary to be the required one to set, and positional parameter means it's a required parameter.
While currently we still want to make sure multichannel is passed in even it's a kwarg (per doc it's required) @kasobol-msft @annatisch please let me know if you have any thought!